packages/cosmic-swingset/README.md

```diff
@@ -23,10 +23,9 @@ In any case, for now, you will be needing to build the solo node from the source
 
 If you want to build and install from sources, you need to install
 - [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- [node.js](https://nodejs.org/en/) (you need at least version 11). This installs 2 binaries named `node` (JavaScript runtime) and `npm` (JavaScript package manager)
```

Comment:
Yes, please remove as much of this as is duplicated by (and point to) the documentation site.

Reply:
Done.